javascript/javascript3/week1/city.js

```diff
@@ -0,0 +1,62 @@
+
+const apiKey = "06150efd3d284c694f2952a08be1740b";
+const main = document.getElementById('main');
+const form = document.getElementById('form');
+const search = document.getElementById('search');
+const button = document.getElementById('myBtn');
+const url = (city) => `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}`;
+async function getWeatherByLocation(city) {
+  const resp = await fetch(url(city), {
+    origin: "cros"
+  });
+  const respData = await resp.json();
+  addWeatherToPage(respData);
+}
+
+function addWeatherToPage(data) {
+  const temp = Ktoc(data.main.temp);
+  const speed = (data.wind.speed);
+  const sunrise = new Date((data.sys.sunrise) * 1000).toLocaleTimeString();
+  const sunset = new Date((data.sys.sunset) * 1000).toLocaleTimeString();
+  let latitude = (data.coord.lat);
```

Comment:
The issue with `let` variables is that they can be re-defined later in your code. If you don't plan to re-define your variable, always use `const`, because it will make sure you don't accidentally re-defined your variable. 🙂